### ISSUE TITLE
WBIC to NHD crosswalk doesn't use the full hydrolayer

### ISSUE THREAD

**Issue by jordansread** [USGS-R/necsc-lake-modeling#92]
It uses the subset hydrolayer that only has ~2600 features.

**Comment by jordansread** [comment]
https://github.com/USGS-R/necsc-lake-modeling/blob/master/scripts/crosswalk_nhd_wbic.R#L5

**Comment by jordansread** [comment]
I did this locally, but we need to capture pulling down the source of the full hydrolayer or at least note it in the comments.

**Comment by lawinslow** [comment]
Hmmm, so what is the issue? It's linking to the subset of the lakes we model, right?

**Comment by lawinslow** [comment]
You mean capture downloading NHD?

https://github.com/USGS-R/necsc-lake-modeling/blob/master/remake_nhd.yml#L26

**Comment by jordansread** [comment]
no, that shapefile was based on WiLMA version 1, so it only has 2600 lakes. We model (I think) more than 3000 WI lakes. 

And ☝️ I was talking about downloading the full WI hydrolayer for mapping. I ran your `rgdal::over` analysis against my copy of the full hydrolayer locally, but don't have the code up here because I didn't want to check the huge shapefile into the repo.

**Comment by lawinslow** [comment]
Ah, ok. makes sense. If you want to solve #93 then the SWIMMS depth data linking will need to be fixed as well once the updated crosswalk is done. 

https://github.com/USGS-R/necsc-lake-modeling/blob/master/scripts/depth_link_swimms.R#L6

**Comment by jordansread** [comment]
yep
